src/Seeds/Data/cameras.json

```diff
@@ -1,6 +1,17 @@
-[{
-  "id": "default",
-  "latitude": "53.27137",
-  "longitude": "-7.827415"
-}
+[
+  {
+    "id": "default",
```

Comment:
Is adding more cameras to the seed really necessary? If yes, maybe we should use different values for our entries.

Reply:
done

Reply:
After talking with @danidoff  we decided that we should keep just one camera inside of our seed, so the comment I left becomes invalid. Let's make this change and then we're good.

Reply:
done